Modules/Files/src/Files.Domain/Entities/FileMetadata.cs

```diff
@@ -45,6 +45,17 @@ public FileMetadata(IdentityAddress createdBy, DeviceId createdByDevice, Identit
     public long CipherSize { get; set; }
     public byte[] CipherHash { get; set; }
 
+    public byte[]? Content { get; private set; }
```

Comment:
The `Content` is theoretically not nullable. The only case it could be null is when it's not loaded. But I think we shouldn't leak this persistence detail to the domain model. 
So better remove the `?`. I don't like that, but I think I'd still prefer it compared to the alternative.